README.md

```diff
@@ -103,8 +103,8 @@ p.readFrom(Sources.fileWatcher(path))
 jet.newJob(p).join();
 ```
 
-Before running this make an empty directory and point the `path`
-variable to it. While the job is running copy some text files into it
+Before running this, make an empty directory and point the `path`
```

Comment:
This comma is not needed.

Reply:
Did you verify this with a native?
I'm quite convinced that if you reverse the order of `Make an empty directory and point the `path` variable to it before running this`, it deserves a comma (similar with above examples)

Reply:
The difference is whether it's a noun phrase or a subordinate clause. `Before running this` is not a subordinate clause, but `before you run this` would be.